stream.go

```diff
@@ -177,7 +177,6 @@ func (stream *Stream) WriteEmptyObject() {
 func (stream *Stream) WriteMore() {
 	stream.writeByte(',')
 	stream.writeIndention(0)
-	stream.Flush()
```

Comment:
LGTM, except this line. Not sure what it's here for and if it could be removed. Pls have a check @taowen